src/shogun/mathematics/Statistics.cpp

```diff
@@ -573,9 +574,7 @@ float64_t CStatistics::erfc8_weighted_sum(float64_t x)
 
 float64_t CStatistics::normal_cdf(float64_t x, float64_t std_dev)
 {
-	SG_SERROR("NOT IMPLEMENTED");
-	return 0;
-//	return 0.5*(error_function_complement(-x/std_dev/1.41421356237309514547));
```

Comment:
@lambday 
I found a bug when I worked on Gaussian processes.
These inputs are used to re-produce the bug.